doc/dev/crimson/pipelines.rst

```diff
@@ -0,0 +1,258 @@
+=========
+Pipelines
+=========
+
+RADOS Protocol Requirements
+===========================
+
+Requests
+--------
+
+OSD serves different kinds of requests. Some of them are targeting the local state of the connected
+OSD instance. For instance, the ``ceph config get`` command is directed straight to the local config
```

Comment:
I don't think it's useful to mention ceph config get commands here.  There are actually a ton of different queries and debug interfaces for the osds, clients, monitors, etc that all have different rules.  I think you should avoid discussing anything other than librados reads and writes.

Reply:
ack. removed.